Packs/DateTime_To_ADFileTime/Scripts/DateTimeToADTime/DateTimeToADTime.py

```diff
@@ -0,0 +1,67 @@
+from calendar import timegm
+from datetime import datetime, timedelta, tzinfo
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+args = demisto.args()
+days_ago = args['days_ago']
+
+
+# http://support.microsoft.com/kb/167296
+# How To Convert a UNIX time_t to a Win32 FILETIME or SYSTEMTIME
+EPOCH_AS_FILETIME = 116444736000000000  # January 1, 1970 as MS file time
+HUNDREDS_OF_NANOSECONDS = 10000000
+
+
+ZERO = timedelta(0)
+HOUR = timedelta(hours=1)
+
+
+class UTC(tzinfo):
+    """UTC"""
+
+    def utcoffset(self, dt):
+        return ZERO
+
+    def tzname(self, dt):
+        return "UTC"
+
+    def dst(self, dt):
+        return ZERO
+
+
+utc = UTC()
+
+
+def dt_to_filetime(dt):
+    """Converts a datetime to Microsoft filetime format. If the object is
+    time zone-naive, it is forced to UTC before conversion.
+    >>> "%.0f" % dt_to_filetime(datetime(2009, 7, 25, 23, 0))
+    '128930364000000000'
+    >>> dt_to_filetime(datetime(1970, 1, 1, 0, 0, tzinfo=utc))
+    116444736000000000L
+    >>> dt_to_filetime(datetime(1970, 1, 1, 0, 0))
+    116444736000000000L
+    """
+    if (dt.tzinfo is None) or (dt.tzinfo.utcoffset(dt) is None):
+        dt = dt.replace(tzinfo=utc)
+    # return EPOCH_AS_FILETIME + (timegm(dt.timetuple()) * HUNDREDS_OF_NANOSECONDS)
+    adTime = EPOCH_AS_FILETIME + (timegm(dt.timetuple()) * HUNDREDS_OF_NANOSECONDS)
+    adTimeStr = str(adTime)
+    demisto.results({
+        "Type": entryTypes["note"],
+        "ContentsFormat": formats["json"],
+        "Contents": adTime,
+        "HumanReadable": adTime,
+        "EntryContext": {"ADFileTime": adTime, "ADFileTimeStr": adTimeStr}
+    })
+
+
+new_date = datetime.today() - timedelta(int(days_ago))
+dt = new_date.replace(hour=0, minute=0, second=0, microsecond=0)
+
+
+nt_time = dt_to_filetime(dt)
```

Comment:
` dt_to_filetime(dt)` does not return anything, so the assignment is meaningless.